src/API/Base.php

```diff
@@ -129,6 +130,7 @@ public static function make_request( $endpoint, $method = 'POST', $payload = arr
 	 * @return array
 	 *
 	 * @throws \Exception PHP exception.
+	 * @throws \PinterestApiException PHP exception.
```

Comment:
This should match the import at the top of the file.

```suggestion
	 * @throws ApiException PHP exception.
```